### PR TITLE
docs: improve  readme

### PR DESCRIPTION
**Pull request by trim21**

### CODE REVIEW

File: docs/zh_CN/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-MinIO Javascript使用[gulp](http://gulpjs.com/)来管理它的依赖。
-
 ```bash
 $ git clone https://github.com/$USER_ID/minio-js
 $ cd minio-js
 $ npm install
-$ gulp
+$ npm run lint
+$ npm run format
+$ npm run test
```
Comment (prakashsvmx) [inline docs/zh_CN/CONTRIBUTING.md:10]: this change  needs to be updated to `CONTRIBUTING.md` at root level ( english)

File: README.md [hunk 2]
```diff
@@ -26,9 +26,7 @@ npm install -g
 
 ## Using with TypeScript
 
-```sh
-npm install --save-dev @types/minio
-```
+`minio>7.1.0` is shipped with builtin type definition, `@types/minio` is no longer need.
```
Comment (aldy505) [inline README.md:29]:
```suggestion
`minio>7.1.0` is shipped with builtin type definition, `@types/minio` is no longer needed.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -4,10 +4,30 @@ Fork [minio-js upstream](https://github.com/minio/minio-js/fork) source reposito
 ```bash
 $ git clone https://github.com/$USER_ID/minio-js
 $ cd minio-js
+```
+
+### Install npm deps
```
Comment (aldy505) [inline CONTRIBUTING.md:9]:
```suggestion
### Install npm dependencies
```

I prefer things to be a bit formal here

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -4,10 +4,30 @@ Fork [minio-js upstream](https://github.com/minio/minio-js/fork) source reposito
...
+
+```bash
 $ npm install
-$ npm test
-$ npm build
-...
+```
+
+### format code (with prettier)
```
Comment (aldy505) [inline CONTRIBUTING.md:15]:
```suggestion
### Format code (with prettier)
```

**Review by prakashsvmx — APPROVED** [review]

**Pull request merged**